app/javascript/components/util/InfoTooltip.js

```diff
@@ -131,6 +131,17 @@ const TOOLTIP_TEXT = {
     </div>
   ),
 
+  blockedSMS: (
+    <div>
+      The owner of this phone number has <br></br>
```

Comment:
Instead of `<br></br>`, you can actually just do `<br />` (there are plenty of debates about that, but `<br />` is what we use around the app so let's keep it consistent). I'm guessing this is fixing the weird cutoff thing we were seeing?